### ISSUE TITLE
[nice2have] to see to which gateway device is connected

### ISSUE THREAD

**Issue by Masterz69** [AlexxIT/XiaomiGateway3#525]
As far I see:
- Attribute ```parent``` in ```zigbee``` entity contains router NWK when device connected to router.
- If device connected to coordinator/gateway directly - ```parent``` is empty ('-').

This is enough to build table or even map in case of one/single gateway.

In case off multiple gateways used - would be nice to have information: to which gateway device is connected/belongs.
F.e. by using same ```parent``` attribute.

**Comment by AlexxIT** [comment]
Its possible: https://github.com/AlexxIT/XiaomiGateway3/releases/tag/v1.2.4b3
But in 2.0-alpha template broken

**Comment by Masterz69** [comment]
Thanx alot. Will wait till fixed in 2.0.

**Comment by AlexxIT** [comment]
New attributes_template will be in 2.0 version

```
xiaomi_gateway3:
  attributes_template: |
    {{{
      "device_name": device.info.name,
      "device_fw_ver": device.fw_ver,
      "device_model": device.model,
      "device_market_model": device.info.model,
      "gateway_name": gateway.info.name,
      "gateway_fw_ver": gateway.fw_ver
    }}}
```
